### PR TITLE
Introduce `pip-tools` to help maintain packages [#618]

### PR DESCRIPTION
**Pull request by marksweb**
This follows on from #620 - starting from an upto date main branch & keeping these changes as simple as they should be.

There is a bug fix release for django which would be worth introducing with this - [2.0.13](https://docs.djangoproject.com/en/3.2/releases/2.0.13/)

### CODE REVIEW

**Comment by marksweb** [comment]
@carltongibson This is as simple as the change should have been...

**Review by carltongibson — COMMENTED** [review]
OK, so in general I think we can adopt this... but at the speed @amakarudze is happy with, because it's a new tool that she has to learn. 

@marksweb — I think `pip-tools` is perhaps not standard, so could you add paragraph to the README about it, with a link to the docs, and the core _Adjust `.in`, run `pip-compile`_ flow, so that future maintainers have a guide?

File: requirements.in
Comment (carltongibson) [inline requirements.in:20]: So this is the key one. Getting back onto a supported Django would be a good step. 

But yes, we can bump Django 2.0 to latest. 👍

Comment (marksweb) [reply requirements.in:20]: I saw some talk of people running django-suit from forks which have support for django 3 so there's a potential way to get django updated that way while replacing the functionality django-suit brings.

https://github.com/darklow/django-suit/issues/475#issuecomment-856635914

**Comment by marksweb** [comment]
> OK, so in general I think we can adopt this... but at the speed @amakarudze is happy with, because it's a new tool that she has to learn.
> 
> @marksweb — I think `pip-tools` is perhaps not standard, so could you add paragraph to the README about it, with a link to the docs, and the core _Adjust `.in`, run `pip-compile`_ flow, so that future maintainers have a guide?

Yeah completely agree, but it's key to get this involved while there isn't much movement in the requirements, if the upcoming goals are to move packages on. And I'm happy to support @amakarudze with all this.

I certainly can. I've just done very similar for `django-cms` where I added this to the docs requirements. 👍

**Comment by carltongibson** [comment]
Thanks @marksweb — your help here is very much appreciated.

**Comment by marksweb** [comment]
Useful info added to the readme 👍

**Review by carltongibson — APPROVED** [review]
That's a nice description. Thanks @marksweb 🏅

**Comment by marksweb** [comment]
@amakarudze So this is ready to merge when you are.

As it stands, merging this won't change anything other than installing `pip-tools` and a patch release for a django regression.

Once you've merged this, run `pip-compile -U` to get a feel for what this will do. You'll see all the dependencies update, things like boto.

Most upgrades, when there are as many packages pinned as we have here, will be safe, patch releases and minor version changes.

If you've got any questions about anything, let me know.

**Comment by amakarudze** [comment]
Sorry, I had thought I merged this but have been experiencing some network issues so would get many errors on GitHub. Let me merge now and then do deployment tomorrow.

**Pull request merged**
